.github/workflows/sync_docs.yml

```
@@ -82,6 +67,34 @@ jobs:
             ${{ runner.os }}-website-
       - run: yarn install --frozen-lockfile
         working-directory: pantsbuild.org
+
+      # Make a new version, if required
+      - name: Make a new version
+        if: endsWith(inputs.version, "a0")
+        run: |
+          BASE_VERSION=$(echo "${{ inputs.version }}" | sed -E 's/^([0-9]+)\.([0-9]+)\.\w+.*/\1.\2/')
```

I don't know if we like this trick, but I think this could be the following, which is shorter if nothing else:

```suggestion
          BASE_VERSION=$(echo "${{ inputs.version }}" | cut -f1,2 -d.)
```

---

I'll leave this one out since I'm doing the below, and therefore won't need to touch this line